src/main/scala/com/github/tarao/namedcap/MultiMap.scala

```diff
@@ -1,7 +1,7 @@
 package com.github.tarao.namedcap
 
 import scala.collection.immutable.{Map => ImmutableMap}
-import scala.collection.Map
+import scala.collection.{Map, Seq}
```

Comment:
https://docs.scala-lang.org/overviews/core/collections-migration-213.html#scalaseq-varargs-and-scalaindexedseq-migration

> In Scala 2.13 scala.Seq[+A] is an alias for scala.collection.immutable.Seq[A], instead of scala.collection.Seq[A]

So, I chose to explicitly use `scala.collection.Seq`.